### DISCUSSION TITLE
Bu ekibin yaptığı projeleri toparlayıp bir platform haline getirerek afet veya afet dışı zamanlarda hazırlıklı olmak

### DISCUSSION THREAD

**Discussion by kayisyaparemin** [acikkaynak/deprem-yardim-projesi#46] [Ideas]
selam arkadaşlar,

Kısa zamanda çok güzel projeler yapıldı. Discussion'dan sonra görüyoruz ki hala fikirler çıkıyor çıkacak da. Benim fikrim ise bu fikirleri genel bir platform altında toparlayarak afet zamanlarında veya afet yaşanmadığı durumlarda:

- afetler hakkında bilinç oluşturmak
- afet eğitimleri düzenlemek
- afet durumunda koordinasyonu sağlamak için hazırlık yapmak
- afetten etkilenenler ile yardımcı olmak isteyenleri buluşturmak vb. gibi amaçları yerine getirmek.
Örnek olarak bu discussion bölümünde gördüğüm bir fikir olan afet bölgelerinde acil durumlarda kim hangi görevi yapabilir şeklinde bir veriseti oluşturalım projesini hayata geçirip bu platformdan afet tatbikatı yapabiliriz. Aynı şekilde bu uygulama afet anında Twitter'dan tweetlerden koordine olmaya çalışmak yerine ilk akla bu platform gelecek şekilde koordine olmak için kullanabiliriz. Bu şekilde gelen bir kullanıcı aynı platformdan afet haritasını inceleyebilir. Aynı şekilde afet anında ilk adres olarak buraya gelen kullanıcı yardım isteyebilir veya yardımcı olabilir. Bir şekilde afet ile ilgili akla gelen her şey için başvurabileceğimiz tek yer olması gerektiğini düşünüyorum. Sağolsunlar bu ekipten önce aklımıza gelen ilk yer twitter'dı. Artık böyle olmasın. 
Yine bu discussion'da çocuklara ve gençlere mentorluk / psikolojik yardım fikirleri görüyorum. Bu ve benzeri fikirleri yine bu platformda topladığımız takdirde ve bu platformu ahbap gibi platformlarla beraber yürüttüğümüzde afet sonrasında yapılabilecekler şeklinde diğer tüm fikirleri barındırabileceğini düşünüyorum.
Bu uzun soluklu bir proje hemen hayata geçmesini tabii ki düşünmüyorum fakat fikirlerinizi merak ediyorum ve yorumlarınızı bekliyorum.

- -

**Comment by alihansencan** [comment]
Selam, fikirlerini cok degerli buldum. Backend konusunda yardimci olabilirim. Konuyu daha detayli konusalim.

**Comment by sercangoger** [comment]
@kayisyaparemin  bütün projeler deprem.io altında toplanıp public harita ve diğer sayfaları herkesin görmesi ve sadece STK ,güvenlik güçlerinin görmesi gereken sistemleri geliştirip tek çatı altında toplamak çok dogru ve yaararlı olacaktır.

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Bu talebiniz Legal ekibin kontrolü sonrasında şu notlar iletildi.
"Legal anlamda sorun olmamakla birlikte gelecek süreçte dernekleşilmesi akabinde proje oluşturulabilir. Aslında Ahbap vb STK lar bu yönde çalışmalarda bulunuyor."

Ek olarak,
Projeler hakkında bilgiye buradan ulaşabilirsiniz: https://afet.org/
